.github/CHANGELOG.md

```diff
@@ -10,6 +10,13 @@
 
 ### Improvements
 
+* PennyLane-Lightning can now be installed without compiling its C++ binaries and will fall back
+  to using the `default.qubit` implementation. Skipping compilation is achieved by setting the
+  `SKIP_COMPILATION` environment variable, e.g., `export SKIP_COMPILATION=True`. This feature
```

Comment:
Just wondering, should we also list Windows env var rules?
```suggestion
  `SKIP_COMPILATION` environment variable, e.g., Linux/MacOS: `export SKIP_COMPILATION=True`, Windows: `set SKIP_COMPILATION=True`. This feature
```

or is it fine to just list Linux?

Reply:
Good point, forgot about windows!